tests/tests_integration/test_api/test_events.py

```diff
@@ -215,7 +215,7 @@ def test_aggregate_has_type(self, cognite_client: CogniteClient, event_list: Eve
 
         count = cognite_client.events.aggregate_count(EventProperty.type, advanced_filter=is_integration_test)
 
-        assert count >= sum(1 for e in event_list if e.type)
+        assert count >= sum(bool(e.type) for e in event_list)
```

Comment:
Perhaps using `len` makes it more readable? (I know sum was used in the original) I leave it up to you
```suggestion
        assert count >= len([e in event_list if e.type])
```
If you decide to change it, there's also this one later:
`assert count >= sum(s.asset_id is not None for s in sequence_list)`

Reply:
I am kind of 50-50 on this. 🤷‍♂️  Won't calculating len on an iterable increase cost compared to a generator? Readability and performance are both important. Maybe a comment to clarify?

Reply:
Honestly, this is a test, and readability matters most. I think the main issue with your approach is the `bool(e.type)` which frankly looks weird to the untrained eye. I think I favor the original one, or `len`, just for its clarity.

If this had been a performance-critical part of the SDK, then I'd favor whatever would run the fastest 😅 But yeah, let's keep in mind this is after all just a test.

Reply:
Fair enough, I think the tests take long enough already, but I guess developers are not running them that often, so probably good to prioritise readability here. 👍